.github/workflows/image-publish.yml

```diff
@@ -79,16 +80,21 @@ jobs:
           elif [[ ${{ env.BRANCH_NAME }} =~ ^7\.[0-9]+\.[0-9]+$ ]]; then
             docker buildx build --platform linux/amd64 -f build/dockerfiles/assembly.Dockerfile --push -t quay.io/che-incubator/che-code:${{env.BRANCH_NAME}}  . 
           fi
+  release-finish:
+    name: release-finish
+    needs: [build, assembly]
+    runs-on: ubuntu-20.04
+    if: (success() || failure()) && github.ref != 'refs/heads/main'
```

Comment:
is there another status beside success and failure? if not, then can we simplify and do:
```
    if: github.ref != 'refs/heads/main'
```

Reply:
I dont need this to be run on cancelled builds